Lib/ensurepip/__init__.py

```diff
@@ -104,16 +157,21 @@ def _bootstrap(*, root=None, upgrade=False, user=False,
         # Put our bundled wheels into a temporary directory and construct the
         # additional paths that need added to sys.path
         additional_paths = []
-        for project, version, py_tag in _PROJECTS:
-            wheel_name = "{}-{}-{}-none-any.whl".format(project, version, py_tag)
-            whl = resources.read_binary(
-                _bundled,
-                wheel_name,
-            )
-            with open(os.path.join(tmpdir, wheel_name), "wb") as fp:
+        for name, package in _get_packages().items():
+            if package.wheel_name:
```

Comment:
This is confusing. What about:

```suggestion
            # bundled wheels have names:
            if package.wheel_name:
```

Reply:
I added comments inside, is it better?

Reply:
Yes.